include/kernels/clover_trace.cuh

```diff
@@ -157,7 +186,7 @@ namespace quda {
     __device__ __host__ inline void operator()(int x_cb)
     {
       // odd parity
-      cloverSigmaTraceCompute<Arg>(arg, x_cb, 1);
+      cloverSigmaTraceCompute<Arg>(arg, x_cb, 0);
```

Comment:
should I try to make this a parameter that one can select from interface_quda.cpp ?

Reply:
> should I try to make this a parameter that one can select from interface_quda.cpp ?

Sounds good to me